### PR TITLE
Pre-Book Club Meeting Slides for Ch 1 and Ch 2

### PR DESCRIPTION
**Pull request by jellisfw**
Initial Content for 2024-07-24 Chapters 1 and 2. This will likely update or have additional data added during the meeting itself.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -9,5 +9,7 @@ Depends:
     R (>= 3.1.0)
 Imports: 
     bookdown,
+    casualdata,
```
Comment (jonthegeek) [inline DESCRIPTION]:
```suggestion
```

This package doesn't appear to exist. Did you add it via a typo?

**Review by jonthegeek — DISMISSED** [review]
I'm approving this (after I commit a change), but I encourage you to make slides fit on one page without scrolling. We'll hopefully convert everything to a quarto presentation relatively soon, and those long slides will be officially weird once we do (the text will go off the bottom of the slide with no way to view it).

**Review by jonthegeek — APPROVED** [review]
Re-approving with typo-package removed.

**Comment by jellisfw** [comment]
@jonthegeek thanks for feedback and approval. For the multiple page slides, when going through the site with this pushed in it, the content appeared all there. Is there a different location that it would cut the slides off. Is "one page" considered as long as I do not have to scroll or is there a set number of lines stated? Thank you

**Comment by jonthegeek** [comment]
> @jonthegeek thanks for feedback and approval. For the multiple page slides, when going through the site with this pushed in it, the content appeared all there. Is there a different location that it would cut the slides off. Is "one page" considered as long as I do not have to scroll or is there a set number of lines stated? Thank you

It won't be cut off today, but it will be in the "true slides" quarto upgrade that's coming soonish. It's hard to say exactly what will fit, because it depends on how much is code, spacing, etc. Just try to keep it feeling "slide like."

HOPEFULLY I can implement the upgrade soon to make it easier to grok what I'm talking about 😊

**Pull request merged**